Core/Keyed/FloatMenu.xml

```diff
@@ -136,6 +138,8 @@
   <InstallImplantAlreadyMaxLevel>уже максимальный уровень</InstallImplantAlreadyMaxLevel>
   <!-- EN: Upgrade {0} to level {1} -->
   <UpgradeImplant>Улучшить {0} до уровня {1}</UpgradeImplant>
+  <!-- EN: violation of faction ally -->
+  <SurgeryViolationFellowFactionMember>надругательство над членом дружественной фракции</SurgeryViolationFellowFactionMember>
```

Comment:
"Violation" у нас уже переведена как механика: `<Intro_Wimp.LetterTextUnauthorizedSurgery.slateRef>Несанкционированная, нарушающая права операция была проведена на {SUBJECT_definite}, госте, которого вы должны были защищать. По причине этого надругательства [asker_pronoun] попытается покинуть ваше поселение. [failLetterEndingCommon]</Intro_Wimp.LetterTextUnauthorizedSurgery.slateRef>`
```suggestion
  <SurgeryViolationFellowFactionMember>нарушение прав члена дружественной фракции</SurgeryViolationFellowFactionMember>
```

Reply:
Строчки из перевода здесь же. 
Because of this violation, [asker_pronoun] will now attempt to leave. 
По причине этого надругательства [asker_pronoun] попытается покинуть ваше поселение. 

Не вижу никаких проблем в использовании более прозаичных номинаций здесь.

Reply:
В том предложение ты не видишь фразу "*этого* надругательства"? Идёт отсылка как раз к нарушению прав.

"Надругательство" - слишком общее слово. А "нарушение прав" - это конкретика, которая нам здесь и нужна.

Reply:
Не знаю, меня вообще концепт "нарушения прав" где-то в жопе мира без какой-либо структуры-гаранта этих прав очень смущает. Подумай просто, какие, к чёрту, **права** на окраинном мире, куда дай Б-г торговый корабль раз в год залетает и нет ни государств, ничего. Буквально в наборе квестов, с которым я сейчас работаю, можно пустить всю группу беженцев на органы, и всем будет насрать. В таких ситуациях люди могут руководствоваться разве что универсальной этикой (максимум моралью внутри общины), и надругательство прекрасно иллюстрирует, что работают не законы (с правовой стороной), а морально-этические ориентиры, с помощью которых мы оцениваем этичность (или моральность) того или иного действия.

Reply:
Речь идёт про квесты от Империи, и работают как раз именно законы.

Reply:
> Речь идёт про квесты от Империи, и работают как раз именно законы.

а точно **только** про них?
Потому что подобный модификатор есть и у членов групп беженцев, которые ни к каким фракциям не принадлежат